examples/tasks/async_compute.rs

```diff
@@ -0,0 +1,121 @@
+use bevy::{prelude::*, tasks::{AsyncComputeTaskPool, Task}};
+use std::time::{Instant, Duration};
+use rand::Rng;
+use futures_lite::future;
+
+/// This example shows how to use the ECS and the AsyncComputeTaskPool
+/// to spawn, poll, and complete tasks across systems and system ticks.
+
+// Number of cubes to spawn across the x, y, and z axis
+const NUM_CUBES: i32 = 6;

```

Comment:
```suggestion
const NUM_CUBES: u32 = 6;
```

since a _negative_ number of cubes doesn't make sense, should probably make this unsigned.